msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md

```diff
@@ -148,7 +157,16 @@ To enable shared meeting stage, configure your app manifest as follows:
   ]
 ```
 
-See how to [design a shared meeting stage experience](~/apps-in-teams-meetings/design/designing-apps-in-meetings.md).
+For more information, see [app manifest](../resources/schema/manifest-schema-dev-preview.md#configurabletabs).
+
+##### Share specific parts of the app to stage
+
+Participants can share specific parts of the app to the collaborative meeting stage by using the share to stage APIs. The APIs are available within the Teams client SDK and are invoked from the app side panel.
+
+
+<img src="../assets/images/apps-in-meetings/share-specific-content-to-stage.png" alt="Share specific parts of the app" width = "900"/>
+
+To share specific parts of the app to stage, you must add the share to stage APIs to Teams client SDK. For more information, see [API reference](API-references.md).
```

Comment:
the "you must add" clause might be a bit confusing, as it might imply one has to add their APIs to the client SDK library. 

Suggestion:
To share specific parts of the app to stage, you must invoked the related APIs in the Teams Client SDK library. For more information...

Reply:
I like that Kemet. Slight correction to what you wrote: 

To share specific parts of the app to stage, you must invoke the related APIs in the Teams Client SDK library. For more information...